resources/js/nova.js

```diff
@@ -27,11 +25,28 @@ Nova.booting((Vue, router) => {
             initialScope: initialScope,
             attachProps: true,
             logErrors: true,
-            integrations: [new Integrations.BrowserTracing()],
             tracesSampleRate: 1.0,
             tracingOptions: {
                 trackComponents: true,
             },
+            integrations: [
+                new Sentry.BrowserTracing(),
+                new Sentry.Feedback({
```

Comment:
would maybe add
- `showBranding: false`

Reply:
Ehh I don't mind it